frontend/src/scenes/insights/insightDataLogic.ts

```diff
@@ -244,17 +244,17 @@ export const insightDataLogic = kea<insightDataLogicType>([
          * This subscription updates the insight for all visualizations
          * that haven't been refactored to use the data exploration yet.
          */
-        response: (response: Record<string, any> | null) => {
+        insightData: (insightData: Record<string, any> | null) => {
             if (!values.isUsingDataExploration) {
                 return
             }
 
             actions.setInsight(
                 {
                     ...values.insight,
-                    result: response?.result,
-                    next: response?.next,
-                    // filters: queryNodeToFilter(query.source),
+                    result: insightData?.result,
+                    next: insightData?.next,
+                    filters: queryNodeToFilter(values.querySource),
```

Comment:
is `queryNodeToFilter` safe for non-insight-viz queries? 

We have the case where you have a `PersonNode` for example, and want `filters` to be `{}` and `query` unchanged